app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryCommentsScreen.kt

```diff
@@ -174,6 +173,9 @@ fun GalleryCommentsScreen(galleryDetail: GalleryDetail, navigator: DestinationsN
         val url = EhUrl.getGalleryDetailUrl(galleryDetail.gid, galleryDetail.token, 0, showAll)
         val detail = EhEngine.getGalleryDetail(url)
         comments = detail.comments
+
+        // Refresh comments of GalleryDetail cache
+        galleryDetail.comments = comments
```

Comment:
How can this solve the issue?
It's the same as https://github.com/FooIbar/EhViewer/blob/d880b30ea92822b446800dbe89acc1ce72b2e75d/app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryCommentsScreen.kt#L164-L166